prep-javascript-objects/main.js

```diff
@@ -0,0 +1,16 @@
+const person = {
+  firstName: 'Ahmad',
+  lastName: 'El-Mowafy',
+  hobby: 'coding',
+};
+
+const fullName = person.firstName + ' ' + person.lastName;
+console.log(fullName);
+
+person.job = 'Neurosurgeon';
+console.log("The person's current job is:", person.job);
+
+person.previousJob = 'Prime Minister';
```

Comment:
I forgot to use bracket notation when adding this property to the `person` object. It should've been written as `person['previousJob'] = 'Prime Minister';`